encryption/encryption.cc

```diff
@@ -440,6 +452,14 @@ Status KeyManagedEncryptedEnv::LinkFile(const std::string& src_fname,
 
 Status KeyManagedEncryptedEnv::RenameFile(const std::string& src_fname,
                                           const std::string& dst_fname) {
+  if (ShouldSkipEncryption(dst_fname)) {
+    Status s = target()->RenameFile(src_fname, dst_fname);
+    if (!s.ok()) {
+      return s;
+    }
+    s = key_manager_->DeleteFile(src_fname);
```

Comment:
No need

Reply:
Check that src_fname is also SkipEncryption, return an invalid argument error if not.